networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobSidecarsByRootMessageHandler.java

```diff
@@ -54,19 +52,19 @@ public class BlobSidecarsByRootMessageHandler
   private static final Logger LOG = LogManager.getLogger();
 
   private final Spec spec;
-  private final UInt64 denebForkEpoch;
+  private final SpecConfigDeneb specConfig;
```

Comment:
I'd name it `specConfigDeneb` (Same int the by range message handler)

Reply:
good catch thanks